trojsten/people/templates/trojsten/people/settings.html

```diff
@@ -33,11 +34,84 @@ <h1>{% trans "Account settings" %}</h1>
             {% include "ksp_login/parts/multiple_forms.html" %}
         </div>
 
+        <div role="tabpanel" class="tab-pane fade" id="schools">
+            {% blocktrans %}
+                <p>Here you can see and edit informations, which schoos you have visited during attending our seminars
+                    along dates, since when the information is valid. According to these informations,
+                    your school is showed in results and camp paticipants list.</p>
+
+                <p>If your school is missing in the list, write us an email and we add it to the list.</p>
+            {% endblocktrans %}
+            <form method="post" action="" class="form-horizontal">
+                {% csrf_token %}
+                {{ school_history_form_set.management_form }}
+                <div id='schools_items_container'>
+                  {% for form in school_history_form_set %}
+                    <div class="form-group{% if form.errors %} has-error{% endif %}">
+                    {% for field in form %}
+                        {% if field.name == "school" %}
+                            {% if field.value %}
+                                <label class="col-md-7 control-label">{{ field.field.choices|choice_text:field.value }}</label>
+                                <input type="hidden" name="{{ field.html_name }}" value="{{ field.value }}"/>
+                            {% else %}
+                                {{ field | bootstrap_inline}}
+                            {% endif %}
+                        {% elif field.name == "start_time" %}
+                            <span class="col-md-2">
+                                {{ field | bootstrap_inline}}
+                            </span>
+                        {% elif field.is_hidden %}
+                            {{ field }}
+                        {% else %}
+                            <span class="col-md-2">
+                                {{ field | bootstrap_inline}}
+                            </span>
+                        {% endif %}
+                        {% for error in field.errors %}
+                            <span class="help-block">{{ error }}</span>
+                        {% endfor %}
+                    {% endfor %}
+                    </div>
+                  {% endfor %}
+                </div>
+                <script type="text/html" id="schools-item-template">
+                    <div id="schools-item-__prefix__" class="form-group">
+                    {% for field in school_history_form_set.empty_form %}
+                        {{ field.errors }}
+                        {% if field.name == "school" %}
+                        <span class="col-md-7">
+                            {{ field | bootstrap_inline}}
```

Comment:
Tu by sa zišlo mať autocomplete. Hral som sa s tým asi hodinu, skúšal som podobné veci ako je v registrácii, ale nepodarilo sa mi to. Viete mi poradiť? Podobne by sa aj na dátum zišiel kalendár ako pri registrácii.